### ISSUE TITLE
Panos modules unusable when passing template_stack parameter

### ISSUE THREAD

**Issue by AmGeorgiou** [PaloAltoNetworks/ansible-pan#398]
When using either the `panos_administrator ` or the `panos_bgp` and specify the `template_stack ` we get the following error (might be applicable to more/all modules that have the `template_stack` parameter):

`"msg": "Param \"template\" is required for Panorama but not specified."`

Example Usage:

```
- name: configure admins
  panos_administrator:
    provider: "{{ provider }}"
    admin_username: "foo"
    admin_password: "bar"
    superuser: True
    template_stack: "test-template-stack"
```

Details about our environment:

```
ansible==2.8
ansible-pan==2.2.1
pandevice==0.11.1
pan-python==0.14.0
```

We think this is caused due to incorrect logic in `PaloAltoNetworks/ansible-pan/module_utils/network/panos/panos.py` - PR to follow shortly.

**Comment by AmGeorgiou** [comment]
#399 solves the above for us (we tested it with `panos_administrator` thus far).

**Comment by shinmog** [comment]
Fixed by #399
